core/cmd/local_client.go

```diff
@@ -312,6 +315,61 @@ func (cli *Client) HardReset(c *clipkg.Context) error {
 	return nil
 }
 
+type HealthCheckPresenter struct {
+	webPresenters.Check
+}
+
+func (p *HealthCheckPresenter) ToRow() []string {
+	red := color.New(color.FgRed).SprintFunc()
+	green := color.New(color.FgGreen).SprintFunc()
+
+	var status string
+
+	switch p.Status {
+	case health.StatusFailing:
+		status = red(p.Status)
+	case health.StatusPassing:
+		status = green(p.Status)
+	}
+
+	return []string{
+		p.Name,
+		status,
+		p.Output,
+	}
+}
+
+type HealthCheckPresenters []HealthCheckPresenter
+
+// RenderTable implements TableRenderer
+func (ps HealthCheckPresenters) RenderTable(rt RendererTable) error {
+	headers := []string{"Name", "Status", "Output"}
+	rows := [][]string{}
+
+	for _, p := range ps {
+		rows = append(rows, p.ToRow())
+	}
+
+	renderList(headers, rows, rt.Writer)
+
+	return nil
+}
+
+// Status will display the health of various services
+func (cli *Client) Status(c *clipkg.Context) error {
+	resp, err := cli.HTTP.Get("/health?full=1", nil)
+	if err != nil {
+		return cli.errorOut(err)
+	}
+	defer func() {
+		if cerr := resp.Body.Close(); cerr != nil {
+			err = multierr.Append(err, cerr)
```

Comment:
hm if it's just one error, you could use `errors.Wrap(cerr,..)` i suppose? or is there a certain difference?

Reply:
I copied this pattern from the other functions in the file (`dropAndCreateDB` etc), I think the idea is that it will merge this error together with the one from `errorOut`/``renderAPIResponse`.